tests/data/decl/coffee-machine.yml

```diff
@@ -1,6 +1,12 @@
 # SPDX-FileCopyrightText: Copyright DB Netz AG and the capellambse contributors
 # SPDX-License-Identifier: Apache-2.0
-
+capellambse: 1.0.0
+model:
+  version: 12345678
+  url: https://example.com/model.git
+referencing: explicit
+generator: Declarative Modelling Generator 1.0.0
+---
```

Comment:
I'm against that, since we also use this example file in `test_decl.py`. For compromise: I've added optional to the metadata section description on the documentation page.